Test failure in org.csstudio.saverestore

```
testGetBaseLevels(org.csstudio.saverestore.git.GitManagerTest)  Time elapsed: 0.024 sec  <<< FAILURE!
java.lang.AssertionError: expected:<base2> but was:<secondBase>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at org.csstudio.saverestore.git.GitManagerTest.testGetBaseLevels(GitManagerTest.java:302)
```

It looks like the ordering in `GitManager.getBaseLevels()` is not reliable.  We've got it failing reliably over here, but it seems to be passing on Travis.

--- a/applications/saverestore/saverestore-plugins/org.csstudio.saverestore.git.test/src/org/csstudio/saverestore/git/GitManagerTest.java
+++ b/applications/saverestore/saverestore-plugins/org.csstudio.saverestore.git.test/src/org/csstudio/saverestore/git/GitManagerTest.java
@@ -294,13 +294,13 @@ public class GitManagerTest {
     public void testGetBaseLevels() throws GitAPIException, IOException {
         List<BaseLevel> baseLevels = manager.getBaseLevels(branch);
         assertEquals(2, baseLevels.size());
-        assertEquals(branchBase, baseLevels.get(0));
-        assertEquals(branchBase2, baseLevels.get(1));
+        assertTrue(baseLevels.contains(branchBase));
+        assertTrue(baseLevels.contains(branchBase2));
         baseLevels = manager.getBaseLevels(secondBranch);
         assertEquals(3, baseLevels.size());
-        assertEquals(new BaseLevel(secondBranch, branchBase), baseLevels.get(0));
-        assertEquals(new BaseLevel(secondBranch, branchBase2), baseLevels.get(1));
-        assertEquals(new BaseLevel(secondBranch, secondBase), baseLevels.get(2));
+        assertTrue(baseLevels.contains(new BaseLevel(secondBranch, branchBase)));
+        assertTrue(baseLevels.contains(new BaseLevel(secondBranch, branchBase2)));
+        assertTrue(baseLevels.contains(new BaseLevel(secondBranch, secondBase)));
     }
 
     @Test

